Library/Formula/lua.rb

```diff
@@ -114,7 +117,7 @@ def install
   def pc_file; <<-EOS.undent
     V= 5.2
     R= 5.2.4
-    prefix=#{HOMEBREW_PREFIX}
+    prefix=#{prefix}
```

Comment:
Probably. I was quite hard against the idea of rewriting `pkg-config` files to point to `opt_xyz` instead of `xyz` but coming around to the merit of it.